README.md

```diff
@@ -3,13 +3,55 @@
 - List 3 reasons why asymptotic analysis may be misleading with respect to
   actual performance in practice.
 
+Asymptotic analysis might be misleading concerning actual performance in practice for three reasons 
+
+One is the ignoring of constant factors that still impact actual performance but are not factored into the algorithm's growth rate.
+
+Two, the hardware that the algorithm is running on will impact actual performance.
+
+Three, Asymptotic analysis is primarily focused on algorithms' behavior when dealing with large input sizes approaching Infinity. This performance may not accurately represent actual performance, as the input sizes are usually much smaller.
```

Comment:
No, the input sizes do not need to approach infinity, they just need to be larger than $n_0$, which could be 1.